packages/react-core/src/demos/examples/DashboardHeader.js

```diff
@@ -123,20 +124,17 @@ export default class DashboardHeader extends React.Component {
             spacer={{ default: 'spacerNone', md: 'spacerMd' }}
           >
             <ToolbarItem>
-              <Button aria-label="Notifications" variant={ButtonVariant.plain}>
-                <AttentionBellIcon />
-              </Button>
+              <Button aria-label="Notifications" variant={ButtonVariant.plain} icon={<AttentionBellIcon />} />
```

Comment:
Can we just use the regular bell icon here? I'm pretty sure this would need to turn red (`variant="attention"`) if the attention bell icon is used.